osu.Game.Rulesets.Catch/Objects/JuiceStream.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using osu.Game.Audio;
+using osu.Game.Beatmaps;
+using osu.Game.Beatmaps.ControlPoints;
+using osu.Game.Rulesets.Catch.UI;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Objects.Types;
+using OpenTK;
+using osu.Framework.Lists;
+
+namespace osu.Game.Rulesets.Catch.Objects
+{
+    public class JuiceStream : CatchBaseHit, IHasCurve
+    {
+        /// <summary>
+        /// Scoring distance with a speed-adjusted beat length of 1 second.

```

Comment:
I think this xmldoc would explain better in the form of `Positional distance that results in a duration of one second, before any speed adjustments.`